trade-finance-manager-ui/server/types/custom-express-request.ts

```diff
@@ -5,22 +5,25 @@ import { Request } from 'express';
  */
 type RequestParams = Record<string, string>; 
 
-type RequestBody = Record<string, string | number> | string;
-
 /**
  * Obtained from the express `core.Query` type
  */
 type RequestQuery = Record<string, string | string[] | undefined>;
```

Comment:
Should remove `undefined` here to be consistent with [FN-1946](https://github.com/UK-Export-Finance/dtfs2/pull/2582/files#diff-de90ed32b79f6e8659ea71d541c861c7eeec451d089df6bb3dfd78c93a635d5c)

Reply:
`core.Query` does include `undefined` so it is actually 1946 which is wrong